ksqldb-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -106,6 +106,11 @@ runScript
     : RUN SCRIPT STRING
     ;
 
+resourceName
+    : identifier
+    | STRING
```

Comment:
This one deserves some careful thought...  

Does this let in anything we wouldn't want?

Reply:
For topic names, no because the original `PRINT TOPIC` defined topic names like this anyway. Schema registry subject names are pretty similar to toptic names, so I don't think there are any problems there either.